Makefile

```diff
@@ -27,7 +27,7 @@ export PATH := $(LOCAL_BIN_PATH):$(PATH)
 export CGO_ENABLED=0
 
 # Details of the model to use:
-model_version:=v0.0.364
+model_version:=v0.0.365
```

Comment:
We usually put this as bump MRs, and document in the CHANGES.md accordingly. See 0.1.410 as example

Reply:
done

Reply:
@Lir10 are you planing to update the CHANGES.md in a followup MR?

Reply:
yep here it is https://github.com/openshift-online/ocm-sdk-go/pull/936